services/web/src/utils/api/request.js

```diff
@@ -46,17 +49,22 @@ export default async function request(options) {
     let message, status, details;
     try {
       const data = await res.clone().json();
-      message = data.error.message;
-      status = data.error.status;
-      details = data.error.details;
+      if (data.error) {
+        message = data.error.message;
+        status = data.error.status;
+        details = data.error.details;
+      }
     } catch (err) {
       message = await res.clone().text();
     }
     throw new ApiError(message || res.statusText, status || res.status, details);
   }
 
   try {
-    return await res.json();
+    const response = await res.json();
+    // Stub to track requests
+    // trackRequest(options, response.data);
```

Comment:
thats pretty smart

Reply:
I thought so too until I realized we could potentially be sending up plain text passwords :)

Don't worry though I'm now stripping out both the request and response bodies. GTM gets zero user-origin data unless the code explicitly says so.

Reply:
also we should enabled it for staging so we can confirm stuff

Reply:
do you mean just to test it out, or...?